README_BenchCLAMP.md

```diff
@@ -73,9 +73,9 @@ An example command below:
    ```
    python -m semantic_parsing_with_constrained_lm.run_exp \
    --config-name semantic_parsing_with_constrained_lm.configs.benchclamp_gpt3_config \
-   --exp-name 'code-davinci-001_calflow_no_context_low_0_2_dev_eval_constrained_bs_5'
+   --exp-name-pattern 'text-davinci-001_calflow_no_context_low_0_2_dev_eval_constrained_bs_5'
```

Comment:
Looks like `--exp-name` is an invalid argument. `--exp-name-pattern` works.

Reply:
good for now. I will fix that arg with final benchclamp export.